### PR TITLE
Add contracts admin tests

### LINKED ISSUE #946: Add tests for Django admin
# What is needed?
Tests for django admin, similar to: https://github.com/safe-global/safe-config-service/blob/8e0f804aa06fb615e50cd05da19799e34cde39fa/src/safe_apps/tests/test_admin.py

# Background
There are no tests for Django admin

### PR DESCRIPTION
**Pull request by hectorgomezv**
### What was wrong?

Admin classes lack from some unit testing. This PR aims to increase the test coverage for ContractAdmin class. After reviewing/merging this PR, it would be nice to cover other admin classes (SafeContractAdmin, TokenAdmin, etc.) in further PRs.

Closes #946

### How was it fixed?

I added some tests to ContractAdmin class, as I said before once this is validated by the team, we could think on increase the coverage to other admin classes also.

@gnosis/safe-services

### CODE REVIEW

**Comment by hectorgomezv** [comment]
recheck

**Comment by hectorgomezv** [comment]
recheck

**Comment by hectorgomezv** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by hectorgomezv** [comment]
recheck

File: safe_transaction_service/contracts/tests/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+from ..models import Contract
+
+
+class ContractAdminTest(TestCase):
+    request_factory = RequestFactory()
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.alfred = User.objects.create_superuser(
```
Comment (Uxio0) [inline safe_transaction_service/contracts/tests/test_admin.py:19]: Maybe `cls.superuser` would be more descriptive than `alfred` 😉

**Review by Uxio0 — APPROVED** [review]

**Review by Uxio0 — CHANGES_REQUESTED** [review]

**Review by Uxio0 — APPROVED** [review]
Nice, now I see the coverage increased

**Pull request merged**